### DISCUSSION TITLE
Can't open the admin page after deploying to netflify

### DISCUSSION THREAD

**Discussion by rachitpant** [tinacms/tinacms#3846] [Help] [answered]
I have a jekyll blog site that i got working with tna cms locally. i can add new posts(collection) using tina cms admin locally. When i added tina CMS , it asked me for clientID and token , and i set this up already.

After deploying to production(netlify), the only change i did was change the site url in tina cloud. What could i be missing.
Here is the build settings in netlify.
<img width="704" alt="Screenshot 2023-04-22 at 11 57 56 PM" src="https://user-images.githubusercontent.com/1474415/233819111-3033c3db-47cc-45dc-a2f3-0ff306566a86.png">

I haven't set any environment variables in netlify and the rest of the site is working perfectly fine in production.

**Comment by kslstn** [comment]
I'm probably equally confused as you are, because the docs are not consistent here. The Jekyll setup pages don't mention env variables, but the [Netlify page does](https://tina.io/docs/tina-cloud/connecting-site/#environment-variables). I did it like mentioned there and got my `/admin` page. That said, I don't think it actually does something that you can't do by logging in to Tina Cloud. I'm under the impression the admin page is only useful when you're running a React site, so you can get a live preview of content changes. But I'd be glad to hear I'm wrong there.

**Reply by rachitpant** [reply]
I think setting environment variables is the just recommended way , not necessarily a requirement. Also `/admin` page is the crux of Tina , and can be used with any framework that tina supports. If you dont see anything in your admin page , other than link to tina cloud , most likely you would have to setup the schema correctly and are missing other setup instructions.

**Comment by jamespohalloran** [comment, marked as answer]
I think the main culprit is your build command. We had an issus with the docs fixed this morning:
https://github.com/tinacms/tina.io/commit/b1df18dd6c19d16bc949797fed79bffbbe077d6c

Your build command should be `yarn tinacms build && ...`, not `yarn tinacms && ...`
